cves/CVE-2013-7024.yml

```diff
@@ -359,7 +395,7 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: 125
 CWE_note:
```

Comment:
I would just put a quick note of what the cwe is and why you chose it, for both of your files

Reply:
The instructions say to only put additional CWEs that apply under the 'CWE_note'